### PR TITLE
PRO-1315 confirm defaults to affirm

### PR DESCRIPTION
**Pull request by stuartromanek**

### CODE REVIEW

File: modules/@apostrophecms/modal/ui/apos/components/AposModalConfirm.vue [hunk 1]
```diff
@@ -166,7 +171,7 @@ export default {
 }
 
 /deep/ .apos-modal__body {
-  padding: 40px 60px;
+  padding: 60px;
```
Comment (stuartromanek) [inline modules/@apostrophecms/modal/ui/apos/components/AposModalConfirm.vue:174]: sorry

File: modules/@apostrophecms/modal/ui/apos/components/AposModalConfirm.vue [hunk 2]
```diff
@@ -121,6 +122,10 @@ export default {
     if (this.content.form) {
       this.formValues = this.content.form.value;
     }
+    // wait to be drawn
+    setTimeout(() => {
```
[the remark this answers is not in the document]
Comment (stuartromanek) [reply modules/@apostrophecms/modal/ui/apos/components/AposModalConfirm.vue:126]: i did try nextTick first and was unsuccessful .. I was in a similar situation w split button's context menu where I wanted to focus the first element and also had to resort to a timeout.
will check again to be sure though

[the remark this answers is not in the document]
Comment (stuartromanek) [reply modules/@apostrophecms/modal/ui/apos/components/AposModalConfirm.vue]: @abea modals now emit a `ready` event that runs over its own `nextTick`, which might have been the timing problem. Feels less janky and is reusable.

**Review by abea — DISMISSED** [review]

**Comment by abea** [comment]
@stuartromanek This should really have a changelog entry

**Review by abea — APPROVED** [review]

**Review by abea — APPROVED** [review]

**Pull request merged**